Assets/MixedRealityToolkit.Services/InputSimulation/Implementation/MouseRotationProvider.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Utility class to manage toggling of mouse rotation and associated features,
+    /// such as cursor visibility/locking
+    /// </summary>
+    /// <param name="wantsJumping">Show the cursor</param>
```

Comment:
>/// <param name="wantsJumping">Show the cursor</param> [](start = 4, length = 54)

Delete this line (I think it was copy pasted from the other wantsJumping location down below)